modules/glossary-of-common-terms-for-rhoai.adoc

```diff
@@ -0,0 +1,85 @@
+:_module-type: REFERENCE
+
+[id='glossary-of-common-terms-for-{productname-short}_{context}']
+= Glossary of common terms for {productname-short}
+
+This glossary defines common terms for {productname-long}.
+
+accelerator:: In high-performance computing, a specialized circuit that is used to take some of the computational load from the CPU, increasing the efficiency of the system. For example, in deep learning, GPU-accelerated computing is often employed to offload part of the compute workload to a GPU while the main application runs off the CPU 
+//Reference: IBM glossary https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x2048370
+
+artificial intelligence (AI):: The capability to acquire, process, create and apply knowledge in the form of a model to make predictions, recommendations or decisions.
+//Reference: https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x3448902
+
+bias detection:: The process of calculating fairness metrics to detect when AI models are delivering unfair outcomes based on certain attributes.
+//Reference:: IBM glossary https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x9721361
+
+cluster storage:: A persistent volume that retains the files and data within a workbench. A workbench has access to one or more cluster storage instances.
```

Comment:
Should be persistent storage, not cluster storage, right?

https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/storage/storage-overview#persistent-storage